static/js/hotkey.js

```diff
@@ -305,6 +306,20 @@ exports.process_enter_key = function (e) {
         return false;
     }
 
+    // This handles when pressing enter while looking at drafts.
+    // It restores draft that is focused.
+    if ($("#draft_overlay").hasClass("show")) {
+        var draft_list = drafts.draft_model.get();
+        if (document.activeElement.parentElement.hasAttribute("data-draft-id")) {
+             var focused_draft = document.activeElement.parentElement.getAttribute("data-draft-id");
+             drafts.restore_draft(focused_draft);
+        } else {
```

Comment:
If no draft is specifically focused and you press enter, should you just restore the most recent draft @timabbott ?